decimal_test.go

```diff
@@ -508,11 +508,17 @@ func TestDecimal_rescale(t *testing.T) {
 }
 
 func TestDecimal_Floor(t *testing.T) {
-	type testData struct {
+	assert := func(input, expected Decimal) {
```

Comment:
I would prefer this be renamed to something a bit more descriptive like `assertFloor` as it's a bit difficult to tell what it's being used for further down in this doc. 

Further, `assert` tends to mean a specific thing in unit tests and calling this specific function `assert` when it's not a more generic assert but rather a specific one for this test means there's a higher likelihood of confusion.

Reply:
Completely agree on this, thanks for extra details about the reasoning.